harpy/downsample.py

```
@@ -32,11 +33,12 @@
 @click.option('--random-seed', type = click.IntRange(min = 1), help = "Random seed for sampling")
 @click.option('-o', '--output-dir', type = click.Path(exists = False), default = "Downsample", show_default=True,  help = 'Output directory name')
 @click.option('-t', '--threads', default = 4, show_default = True, type = click.IntRange(1,999, clamp = True), help = 'Number of threads to use')
+@click.option('--hpc',  type = HPCProfile(), help = 'HPC submission YAML configuration file')
```

Ruff (0.8.2)

Undefined name `HPCProfile`

(F821)